tests/unit/test_repository.py

```diff
@@ -319,6 +319,29 @@ def test_get_file_content(monkeypatch: pytest.MonkeyPatch, repository_client: Cl
     returned_content = repository_client.get_file_content(path=path)
 
     assert returned_content == content
+    mock_github_repository.get_contents.assert_called_once_with(path)
+
+
+def test_get_file_content_branch(monkeypatch: pytest.MonkeyPatch, repository_client: Client):
+    """
+    arrange: given path and branch, Client with a mocked github repository client that returns
+        content
+    act: when get_file_content is called with the path and branch
+    assert: then the content is returned.
```

Comment:
s/then the content/the content

Reply:
The `then` is actually used to make the description without the arrange/act/assert read nicely, i.e., given ... when ... then ..., this is actually used quite a lot in the test docstrings